src/revoked/contractInterface.integration.test.ts

```diff
@@ -1,114 +1,86 @@
 import { constants } from "ethers";
 
+import { isRevoked, isRevokedOnDocumentStore, isRevokedOnTokenRegistry } from "./contractInterface";
```

Comment:
Same comment to put with the verifiers

Reply:
moved under verifiers/documentStoreRevoked